atlp-rwanda/hackers-ec-be
# #187419061 User should be able to update their profile page

**What does this PR do?**
 - Update user's profile setting

**_Description of the task_**
- Registation is complete
- Users should be able to update only their own profile

_**EndPoint**_
1.   `/api/v1/profile` -> method: GET
2. `/api/v1/profile` -> method: PATCH

How to test this locally:

1. Initialise project
    - Clone the repo and navigate to `feat-profile-setting-187419061`
    - Run `npm install` to install all the packages and dependences
    - Run `npm run migrate:reset` to add test files

2. Test with postman
     - in your postman you can check with these endpoints:
               1. `/api/v1/profile` method: GET for viewing your profile
               2. `/api/v1/profile` method: PATCH for updating your profile

3. you can also check in database 

Pivotal tracker id:

    #187419061
